### PR TITLE
feat(longhornctl): add examples to help menu and doc

### PR DESCRIPTION
**Pull request by c3y1huang**
#### Which issue(s) this PR fixes:

Issue longhorn/longhorn#8992

#### What this PR does / why we need it:

Add examples to the help menu and doc.

#### Special notes for your reviewer:

`None`

#### Additional documentation or context

`None`

### CODE REVIEW

File: cmd/remote/subcmd/check.go [hunk 1]
```diff
@@ -31,7 +31,36 @@ func newCmdCheckPreflight(globalOpts *types.GlobalCmdOptions) *cobra.Command {
 		Use:   consts.SubCmdPreflight,
 		Short: "Check Longhorn preflight",
 		Long: `This command verifies your Kubernetes cluster environment. It performs a series of checks to ensure your cluster meets the requirements for Longhorn to function properly.
-These checks can help to identify issues that might prevent Longhorn from functioning properly.`,
+These checks can help to identify issues that might prevent Longhorn from functioning properly.
+
+Example:
```
Comment (derekbit) [inline cmd/remote/subcmd/check.go:36]: cobra.Command has an `Example` field. We can use
```
		Use:   consts.SubCmdPreflight,
		Short: "Check preflight environment",
		Long: `This command verifies your Kubernetes cluster environment. It performs a series of checks to ensure your cluster meets the requirements for Longhorn to function properly.
These checks can help to identify issues that might prevent Longhorn from functioning properly.`,
		Example:  ....
		PreRun: ....
		Run: ....
		PostRun: ....
....
```

Comment (c3y1huang) [reply cmd/remote/subcmd/check.go]: Fixed, thank you!

File: docs/longhornctl_export_replica.md [hunk 2]
```diff
@@ -10,17 +10,40 @@ It provides data recovery capabilities when the Longhorn system is unavailable.
...
 
 To find available replica data directory names, use the following command:
-> longhornctl get replica
+  $ longhornctl get replica
 
 After the export is completed, you can access the exported data at the specified location on the node provided in the output.
 
 To terminate the replica exporter and stop the replica export process, use the 'stop' subcommand with the original command. For example:
-> longhornctl export replica <options> stop
+  $ longhornctl export replica <options> stop
```
Comment (derekbit) [inline docs/longhornctl_export_replica.md:18]: Is it better?
```suggestion
  `longhornctl export replica <options> stop`
```

File: cmd/remote/subcmd/export.go [hunk 3]
```diff
@@ -38,12 +38,30 @@ It provides data recovery capabilities when the Longhorn system is unavailable.
 To perform an export, provide the name of the replica data directory to the --name option.
 
 To find available replica data directory names, use the following command:
-> longhornctl get replica
+  $ longhornctl get replica
```
Comment (derekbit) [inline cmd/remote/subcmd/export.go:41]: Is it better?
```suggestion
  ```
  longhornctl get replica.
  ```
```

Comment (c3y1huang) [reply cmd/remote/subcmd/export.go:41]: I am not sure, using code block makes it easier to read in doc, but might be harder to read in the terminal. I am find if that is preferred.

Screenshot from 2024-07-17 10-28-02

Comment (derekbit) [reply cmd/remote/subcmd/export.go:41]: I see. It is minor. Let's keep it as it is.

**Review by derekbit — APPROVED** [review]

File: cmd/remote/subcmd/export.go (hunk 3 above)
Comment (c3y1huang) [reply cmd/remote/subcmd/export.go:41]: We revisit this again when @longhorn/doc starts to work on https://github.com/longhorn/longhorn/issues/8998.

**Comment by c3y1huang** [comment]
@mergify backport v1.7.x

**Pull request merged**